biojava-core/src/test/java/org/biojava/nbio/core/sequence/loader/GenbankProxySequenceReaderTest.java

```diff
@@ -69,9 +73,59 @@ public GenbankProxySequenceReaderTest(String gi) {
 			{"254839678"}
 		};
 
-		return Arrays.asList(out);
+		return Arrays.asList(accessorIds);
 	}
 
+	/**
+	 * In {@link GenbankProxySequenceReader} there is a check to see if the requested files are already in the temp
+	 * directory before attempting to retrieve them from the remote server. so simply copying the test files to the temp
+	 * directory avoids calling out to the server and hitting a 429 status code from the server which fails the build.
+	 * @throws IOException
+	 */
+	@Before
+	public void copyTestFiles() throws IOException {
+		Collection<String[]> accessorIds = getExamples();
+		for (String[] arr: accessorIds) {
+			copyTestFileToWorkingDirectory(arr[0]+".gb");
+		}
+	}
+
+	/**
+	 * Convenience method for {@link GenbankProxySequenceReaderTest#copyTestFiles()}
+	 * @param filename name of the file to copy from the resource folder
+	 * @throws IOException when something goes wrong with copying the files.
+	 */
+	private void copyTestFileToWorkingDirectory(String filename) throws IOException {
+		String destRoot = System.getProperty("java.io.tmpdir");
+
+		//if the directory does not end with a slash or backslash then add one
+		if(!(destRoot.endsWith("/") || destRoot.endsWith("\\"))){
+			destRoot += destRoot.contains("/")? "/" : "\\";
+		}
+
+		String dest =  destRoot + filename;
+		String src = "org/biojava/nbio/core/sequence/GenbankProxySequenceReader/" + filename;
```

Comment:
I'd rather use the same package name as the class to keep the resources in, i.e.: `org/biojava/nbio/core/sequence/loader`